src/frontend/app/shared/components/list/list-types/cf-services/cf-service-card/cf-service-card.component.html

```diff
@@ -1,4 +1,4 @@
-<app-meta-card class="service-card">
+<app-meta-card class="service-card" [clickAction]="goToServiceInstances">
```

Comment:
Clicking on the doc/support link button opens new window as expected but also navs existing page to service instance table.